### PR TITLE
Fix `feign-jackson` dependency

### PR DESCRIPTION
**Pull request by doncem**
### Change description ###

Update s2s client library in order to fix the clash of `feign-jackson` dependencies. Innocently squeezing in gradle upgrade

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by doncem** [comment]
_Sneaky entry changes_ come together with that fix as `Pair` was from older feign client dependency

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/JurisdictionToUserMapping.java [hunk 1]
```diff
@@ -17,13 +18,14 @@ public void setUsers(Map<String, Map<String, String>> users) {
             .entrySet()
             .stream()
             .map(this::createPair)
```
Comment (KevinJCross) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/JurisdictionToUserMapping.java]: probably should make this createEntry as it initially was... lol

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/JurisdictionToUserMapping.java [hunk 2]
```diff
@@ -17,13 +18,14 @@ public void setUsers(Map<String, Map<String, String>> users) {
...
-            .collect(toMap(Pair::first, Pair::second));
+            .collect(toMap(Entry::getKey, Entry::getValue));
```
Comment (KevinJCross) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/JurisdictionToUserMapping.java]: this is more readable :smile:

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/JurisdictionToUserMapping.java [hunk 3]
```diff
@@ -17,13 +18,14 @@ public void setUsers(Map<String, Map<String, String>> users) {
...
     }
 
-    private Pair<String, Credential> createPair(Map.Entry<String, Map<String, String>> entry) {
+    private Entry<String, Credential> createPair(Entry<String, Map<String, String>> entry) {
         String key = entry.getKey().toLowerCase();
         Credential cred = new Credential(entry.getValue().get("username"), entry.getValue().get("password"));
-        return new Pair<>(key, cred);
+
+        return new AbstractMap.SimpleEntry<>(key, cred);
```
Comment (KevinJCross) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/JurisdictionToUserMapping.java]: hmmm  ... a bit terse in construction but I guess the getKey and getValue out weigh this little bit of terseness

**Pull request merged**